packages/shorebird_cli/lib/src/code_push_client_wrapper.dart

```diff
@@ -615,4 +618,26 @@ aar artifact already exists, continuing...''',
 
     await promotePatch(appId: appId, patchId: patch.id, channel: channel);
   }
+
+  /// Prints an appropriate error message for the given error and exits with
+  /// code 70. If [progress] is provided, it will be failed with the given
+  /// [message] or [error.toString()] if [message] is null.
+  Never _handleErrorAndExit(
```

Comment:
Couldn’t we have handled the upgrade required case in the code_push_client just like the other exceptions and overridden the message there? I think that would have been a less invasive change and would’ve been more consistent with existing custom error handling?

Reply:
We could have, but my understanding of the code push client design was that it was intentionally decoupled from the CLI, so it wouldn't make sense to reference CLI commands there.

Reply:
(and I think it's important that we tell the user explicitly what command they need to run, as we do elsewhere in the CLI)

Reply:
ah yeah misread this earlier, totally makes sense to have this in the CLI 👍